### ISSUE TITLE
Adjust vets-api code to utilize the new "SETTINGS" env variables instead of existing variables

### ISSUE THREAD

**Issue by patrick-black-oddball** [department-of-veterans-affairs/va.gov-team#95566]
**Blocked by**: https://github.com/department-of-veterans-affairs/va.gov-team/issues/97393

As part of the parameter store normalization effort, variable names have been adjjusted. Settings inside vets-api need to be adjusted to utilize the new variable names

https://docs.google.com/spreadsheets/d/1-7PLskyY67b4J1fsk-FADYxnSN1UrmVziFfFIt_1saU/edit?gid=371837002#gid=371837002

Approx 400 values
## Tasks
- [x] change variable names 
- [ ] Removing the old environment variable setting from the 4 env helm charts

## Acceptance Criteria
- [ ] vets-api code is merged into master using the new variable names
- [ ] environment helm charts have old variable settings removed

**Comment by stevenjcumming** [comment]
Task: "Removing the old environment variable setting from the 4 env helm charts" is blocked until the parent helm charts are updated to handing the new environment variables

**Comment by stevenjcumming** [comment]
Tasks 2 is in progress

**Comment by stevenjcumming** [comment]
Moving to a different ticket
